### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by csbasic**
### What?
- Collection of contact form data

### Why?
- It allows visitors of the website to make inquiries.
- The Data will permits that the wants of visitors of the website are curated. 
- That curated data can be managed.

### How? 
- On the contact page, visitors would fill in the fields on the form and submit. Submitted data is stored on the database.

### CODE REVIEW

File: app/Models/Contact.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Contact extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'email'
+    ];
```
Comment (CollinsTatang) [inline app/Models/Contact.php]: - [x] Kindly add the other contact attribute.

File: resources/views/components/layout.blade.php [hunk 2]
```diff
@@ -80,10 +123,12 @@
                   <li class="dropdown has-dropdown">
                       @php
                         $profilePic = request()->user()::find(auth()->id())->profile_pic;
+                        // dd($profilePic);
```
Comment (CollinsTatang) [inline resources/views/components/layout.blade.php]: - [x] Please remove all commented code from your codebase.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @csbasic, 

Good job so far!
There are some issues that you still need to work on to go to the next step but you are almost there!

### **Highlights 🚀  ✔️ ✔️ ** 

- The repo looks professional. ✔️ 
- Git/GitHub flow best practices have been followed ✔️ 

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [x] Please update your pull request description. Check the image below
Screenshot 2024-05-09 at 18 57 25

- [x] Please your commit message should be written in the imperative tense. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: resources/views/components/layout.blade.php (hunk 2 above)
Comment (csbasic) [reply resources/views/components/layout.blade.php]: Comments removed

File: app/Models/Contact.php (hunk 1 above)
Comment (csbasic) [reply app/Models/Contact.php]: The rest of contact fillable added

**Comment by csbasic** [comment]
Instructions given are met.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional. :heavy_check_mark:
- Git/GitHub flow best practices have been followed :heavy_check_mark:
- You have descriptive PR :heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**